.github/workflows/check-deps.yml

```diff
@@ -0,0 +1,32 @@
+name: Check dependencies versions
+
+on:
+  # Run whenever a pull request is updated
+  pull_request:
+    branches:
+      - main
+jobs:
+  syncpack:
+    runs-on: ubuntu-latest
+    name: Check dependencies
+    steps:
+      # Check out the branch so we can read/write its files
+      - uses: actions/checkout@v3
+      # Use Node.js as syncpack is written in that
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - uses: JamieMason/syncpack-github-action@0.2.2
+        with:
+          package-manager: "pnpm"
+          bot-username: "Syncpack Bot"
+          commit-message: "format and sync package.json files"
+          version-mismatches: "lint"
+          semver-range-mismatches: "ignore"
+          lockfile-mismatches: "fix"
+          format-mismatches: "fix"
```

Comment:
its not commiting so its not true (it will not fix anything). Keep it ignored, and we can have pre-commit to format things